planning/behavior_path_planner/src/scene_module/lane_change/lane_change_module.cpp

```diff
@@ -44,36 +44,54 @@ namespace behavior_path_planner
 {
 using autoware_auto_perception_msgs::msg::ObjectClassification;
 
+#ifdef USE_OLD_ARCHITECTURE
 LaneChangeModule::LaneChangeModule(
   const std::string & name, rclcpp::Node & node, std::shared_ptr<LaneChangeParameters> parameters)
 : SceneModuleInterface{name, node},
   parameters_{std::move(parameters)},
-  rtc_interface_left_(&node, "lane_change_left"),
-  rtc_interface_right_(&node, "lane_change_right"),
   uuid_left_{generateUUID()},
   uuid_right_{generateUUID()}
 {
+  rtc_interface_left_ = std::make_shared<RTCInterface>(&node, "lane_change_left");
+  rtc_interface_right_ = std::make_shared<RTCInterface>(&node, "lane_change_right");
   steering_factor_interface_ptr_ = std::make_unique<SteeringFactorInterface>(&node, "lane_change");
 }
+#else
+LaneChangeModule::LaneChangeModule(
+  const std::string & name, rclcpp::Node & node,
+  const std::shared_ptr<LaneChangeParameters> & parameters,
+  const std::shared_ptr<RTCInterface> & rtc_interface_left,
+  const std::shared_ptr<RTCInterface> & rtc_interface_right)
+: SceneModuleInterface{name, node},
+  parameters_{parameters},
+  rtc_interface_left_{rtc_interface_left},
+  rtc_interface_right_{rtc_interface_right},
+  uuid_left_{generateUUID()},
+  uuid_right_{generateUUID()}
+{
+  steering_factor_interface_ptr_ = std::make_unique<SteeringFactorInterface>(&node, "lane_change");
+}
+#endif
 
 void LaneChangeModule::onEntry()
 {
+  std::cerr << "Lane change on entry.\n";
   RCLCPP_DEBUG(getLogger(), "LANE_CHANGE onEntry");
-  current_state_ = BT::NodeStatus::SUCCESS;
+  current_state_ = ModuleStatus::SUCCESS;
```

Comment:
It should be `IDLE` in the new manager.

Reply:
fixed in 75f758be2a223fb76abb98d9ce966f32cf495a8e